microsoft-edge/webview2/reference/win32/0-9-538/webview2-idl.md

```diff
@@ -3,7 +3,7 @@ description: Host web content in your Win32 app with the Microsoft Edge WebView2
 title: Microsoft Edge WebView2 for Win32 apps
```

Comment:
Can you update the page title please?

Reply:
The title is generated from a template of ours for all the reference docs. I will open a task in our ADO backlog to address this. But will not gate this PR based on this feedback